website/docs/reference/resource-configs/trino-configs.md

```diff
@@ -23,7 +29,24 @@ You can use Starburst/Trino table properties to configure how you want your data
 
 For details on what's supported for each supported data source, refer to either the [Trino Connectors](https://trino.io/docs/current/connector.html) or [Starburst Catalog](https://docs.starburst.io/starburst-galaxy/catalogs/).
 
-For example, this configures the data to materialize as a [Parquet](https://spark.apache.org/docs/latest/sql-data-sources-parquet.html) table with partitions: 
+
+
+### Hive Catalog
+
+At target cataog that uses the Hive connector and a metastore service (HMS) is very typical when working with Starburst and dbt. The following settings are recommended for working with dbt. The intent is to ensure that dbt can perform the frequently executed `DROP` and `RENAME` statements.
```

Comment:
Also .. this should probably be expanded to talk about Hive, Iceberg, and Delta Lake